.github/workflows/build.yml

```
@@ -98,7 +98,11 @@ jobs:
 
       - name: Version Test
         run: |
-          sudo apt install libsecret-1-0
+          sudo apt install libsecret-1-0 dbus-x11 gnome-keyring
+          eval $(dbus-launch --sh-syntax)
+          
+          eval $(echo -n "" | /usr/bin/gnome-keyring-daemon --login)
+          eval $(/usr/bin/gnome-keyring-daemon --components=secrets --start)
```

```suggestion
          dbus-launch --sh-syntax
          
          echo -n "" | /usr/bin/gnome-keyring-daemon --login
          /usr/bin/gnome-keyring-daemon --components=secrets --start
```

I don't believe `eval` is needed here. It is normally used to evaluate a command that is stored in a variable as a string like
```bash
var="ls -alth"
eval $var
```

---

I'll give it a try and see if it is still happy with it!

---

It's erroring again, I'm going to add the `eval`'s back

---

Weird. Ok, we can keep the `eval`'s then.

---

At the least `eval $(dbus-launch --sh-syntax)` is necessary. The application outputs a set of env variables about the currently running dbus session.

I don't believe the others are as there is no output.